snowshu/core/printable_result.py

```diff
@@ -29,41 +29,63 @@ def to_tuple(self) -> list:
                 )
 
 
+def process_relation(graph, relation):
+    sample_size = "N/A"
+    percent = "N/A"
+    percent_is_acceptable = False
+    target_sample_size = "N/A"
+
+    try:
+        deps = len(nx.ancestors(graph, relation))
+        deps = " " if deps == 0 else str(deps)
+
+        if isinstance(relation.population_size, str):
+            target_sample_size = "N/A"
+            percent = "N/A"
+        elif int(relation.population_size) < 1:
+            percent = 0
+            target_sample_size = relation.population_size
+        else:
+            target_sample_size = (
+                relation.population_size
+                if relation.unsampled
+                or (relation.population_size < relation.sampling.size)
+                else relation.sampling.size
+            )
+            percent = int(round(100.0 * (relation.sample_size / target_sample_size)))
+
+        percent_is_acceptable = (
+            True if isinstance(percent, str) else abs(percent - 100) <= 5
+        )
+        sample_size = relation.sample_size
+    except AttributeError as exc:
```

Comment:
Another temporary exception handling, due to JSONDecodeError, relation is missing some attributes causing this function to fail, we handle it so the process doesn't fail, but gives us information in the final table about missing tables.